src/components/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,116 @@
+@import "../../constants/style";
+
+.color-picker {
+  z-index: $menu-item-z-index;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: $spacing--xxs;
+  box-shadow: 0 0 20px rgba($navy--400, 0.12);
+  background-color: $gray--300;
+  list-style: none;
+  padding-left: 0;
+  margin-top: auto;
+  border-radius: $rounded--full;
+
+  > li {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    > button {
+      cursor: pointer;
+      border: none;
+      background: none;
```

Comment:
Imo, not every element needs to have a dedicated classname (and it's also overkill sometimes). So for me it's fine to use css selectors.

Reply:
in this case I'd stick to css selectors just because fitting it into BEM’s naming convention would just mean calling the classes __item or __button instead of having a list item or button css selector, which i don't find too beneficial here tbh

Reply:
while it doesn't make a syntactical difference per se, it's more about consistency among all files for me. Also in the file, the `&`-selector is used as well so it's kind of a mix. But if you think this is more readable, I'm fine with it.

Reply:
fair point, inconsistent naming isn't the way to go